Codeinwp/optimole-wp
# Add PHPStan

### All Submissions:

 

### Changes proposed in this Pull Request:
This adds PHPStan to level 5 and changes have been made to best of my knowledge, so it will be good to have a review to make sure no changes are reverting intended code.

Also, there is one error left in PHPStan which I'm not able to realize why that particular line of code was written the way it is written so feedback there is appreciated.

Closes #566.

### How to test the changes in this Pull Request:

1. Just make sure everything works as before in Optimole plugin.
2. Most importantly to test all the settings about including/excluding.
3. Make sure to have debugging on and make sure there isn't anything in the debug logs.

### Other information:
